src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
can you explain what this is doing a bit more? did you intend to use triple equals?